helpers.js

```diff
@@ -492,16 +495,21 @@ const getNotAnalyzedContracts = (mythXIssuesObjects, contracts) => {
 }
 
 const getArmletClient = (ethAddress, password, clientToolName = 'truffle') => {
+    let isTrial = false;
     const options = { clientToolName };
     if (password && ethAddress) {
         options.ethAddress = ethAddress;
         options.password = password;
     } else if (!password && !ethAddress) {
```

Comment:
There's a corner case here where the user explicitly adds the trial user's access credentials